.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: node_js
+os: linux
+node_js:
+- '9'
+script: echo "Running jobs Install, Build & Test against $(node -v) ..."
+jobs:
+  include:
+  - stage: Install
+    script:
+    - echo "Installing needed modules"
+    - npm install
+  - stage: Build
+    script:
+    - echo "Running build script ..."
+    - npm run-script build
+  - stage: Test
+    script:
+    - echo "Testing branch..." $TRAVIS_BRANCH $TRAVIS_PULL_REQUEST_BRANCH
+    - npm test
+  - stage: Merge to master
```

Comment:
While it's nice to have the option, I don't think we want to merge every PR just because all the tests pass

Reply:
Good point! we can discuss offline.

Reply:
Removed auto merge